packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -390,68 +390,29 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 		return this.runningSummarizer.enqueueSummarize(...args);
 	};
 
-	private async handleSummaryAcks() {
-		let refSequenceNumber = this.runtime.deltaManager.initialSequenceNumber;
-		let ack: IAckedSummary | undefined;
+	/**
+	 * Responsible for receiving and processing all the summaryAcks.
+	 * @param lastAckRefNumber - Before initializing the summarization, if there is a last Ack to be processed, use it.
+	 */
+	private async handleSummaryAcks(lastAckRefNumber: number) {
+		let refSequenceNumber =
+			lastAckRefNumber > 0
```

Comment:
If I understand this, this is saying that if there was an ack processed (by the running summarize) before this function is called, start waiting for ack that is newer that the one that was processed?
Could you please add comments here. It's difficult to follow other wise.